app/views/admin/consumer_apps/_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="form-group">
```

Comment:
how does one deacitave an app? We don't expose the `active` column

Reply:
In the meantime I was able to deactivate the app by removing the authentication key, since it can be emptied. Is that okay?

Reply:
Yup, that's okay. Creators will have full control over their consumer apps, so this would be a valid way for them to "invalidate them" (even though I can't think of a reason why Creators would want to do this). 

The most valuable thing here IMO is the visual feedback for the Creator, so they can understand whether or not an App is operational and also (if that's the case) they can tell if the problem is related with the `auth_key`. This way things aren't broken without any way to understand what is causing it.